Makefile

```diff
@@ -180,6 +180,8 @@ list-versions:
 
 
 dockerbuild-go:
+	# Must remove binaries or they will be copied into the container
+	rm -rf devtools/
```

Comment:
This means we need to re-install after every docker build... better to add this folder to `.dockerignore`

Who approved this PR anyway?! 🤪

Reply:
lol, will open a new issue now and will address this 👍